apps/web/pages/organizations/[organizationId]/invoices/[invoiceId]/index.page.test.tsx

```diff
@@ -27,7 +27,8 @@ jest.mock('next/router', () => ({
 it('should render invoice details', async () => {
   render(<InvoiceDetailsPage />, { wrapper })
 
-  await screen.findByText('INVOICE')
+  const invoiceElements = await screen.findAllByText('INVOICE', { exact: false })
+  expect(invoiceElements).toHaveLength(3)
```

Comment:
Why is it of value 3 here?

Reply:
It finds the invoice title, invoice status with three statuses, and invoice date.